auth_session_timeout/models/ir_config_parameter.py

```diff
@@ -14,23 +14,34 @@ class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
     @tools.ormcache(skiparg=0)
-    def get_session_parameters(self, db):
+    def _auth_timeout_get_parameter_delay(self, db):
         param_model = self.pool['ir.config_parameter']
         cr = self.pool.cursor()
         delay = False
-        urls = []
         try:
             delay = int(param_model.get_param(
                 cr, SUPERUSER_ID, DELAY_KEY, 7200))
+        finally:
+            cr.close()
+        return delay
+
+    @tools.ormcache(skiparg=0)
+    def _auth_timeout_get_parameter_urls(self, db):
+        param_model = self.pool['ir.config_parameter']
+        cr = self.pool.cursor()
+        urls = []
+        try:
             urls = param_model.get_param(
                 cr, SUPERUSER_ID, IGNORED_PATH_KEY, '').split(',')
         finally:
             cr.close()
-        return delay, urls
+        return urls
```

Comment:
This is changing the external facing API. The adapter method would need to return compatibly